bears/documentation/DocumentationStyleBear.py

```diff
@@ -0,0 +1,118 @@
+    CAN_DETECT = {'Documentation'}
+    CAN_FIX = {'Documentation'}
+
+    def run(self, filename, file, language: str,
+            docstyle: str='default', allow_missing_func_desc: str=False):
+        """
+        Checks for certain in-code documentation styles.
+
+        It currently checks for the following style: ::
+
+            The first line needs to have no indentation.
+                - Following lines can have indentation
+
+            :param x:
+                4 space indent
+            :return:
+                also 4 space indent
+                following lines are also 4 space indented
+
+        :param language: The programming language of the file(s).
+        :param docstyle: The docstyle to use. For example ``default`` or
+                         ``doxygen``. Docstyles are language dependent, meaning
+                         not every language is supported by a certain docstyle.
+        :param allow_missing_func_desc: When set ``True`` this will allow
+                         functions with missing descriptions, allowing
+                         functions to start with params.
+        """
+        for doc_comment in extract_documentation(file, language, docstyle):
+            parsed = doc_comment.parse()
+            metadata = iter(parsed)
+
+            # Assuming that the first element is always the only main
+            # description.
+            main_description = next(metadata)
+
+            if main_description.desc == '\n' and not allow_missing_func_desc:
+                warning_desc = """
+Missing function description.
+Please set allow_missing_func_desc:True to ignore this warning.
+"""
+            else:
+                warning_desc = 'Documentation does not have correct style.'
+
+            # one empty line shall follow main description (except it's empty
+            # or no annotations follow).
+            if main_description.desc.strip() != '':
+                main_description = main_description._replace(
+                    desc='\n' + main_description.desc.strip() + '\n' *
+                         (1 if len(parsed) == 1 else 2))
+
+            new_metadata = [main_description]
+            for m in metadata:
+                # Split newlines and remove leading and trailing whitespaces.
+                stripped_desc = list(map(str.strip, m.desc.splitlines()))
+
+                if len(stripped_desc) == 0:
+                    # Annotations should be on their own line, though no
+                    # further description follows.
+                    stripped_desc.append('')
+                else:
+                    # Wrap parameter description onto next line if it follows
+                    # annotation directly.
+                    if stripped_desc[0] != '':
+                        stripped_desc.insert(0, '')
+
+                # Indent with 4 spaces.
+                stripped_desc = ('' if line == '' else ' ' * 4 + line for line
```

Comment:
I do not see tab (`\t`) characters in `bad_file5.py`.

Even ignoring tabs, this bear needs to have an `indent_size` setting, and use that instead of a hard coded `4` here.